plugins/destination/mongodb/go.mod

```diff
@@ -13,6 +13,8 @@ require (
 // TODO: remove once all updates are merged
 replace github.com/apache/arrow/go/v13 => github.com/cloudquery/arrow/go/v13 v13.0.0-20230509053643-898a79b1d3c8
 
+replace github.com/cloudquery/plugin-sdk/v3 => ../../../../plugin-sdk
+
```

Comment:
```suggestion
```

Reply:
This is blocked by https://github.com/cloudquery/plugin-sdk/pull/893